components/create_team/components/team_url/team_url.tsx

```diff
@@ -140,7 +151,7 @@ export default class TeamUrl extends React.PureComponent {
             return;
         }
 
-        const {data, error} = await createTeam(teamSignup.team);
+        const {data, error}: any = await createTeam(teamSignup.team);
```

Comment:
This should be `{data: Team, error: Client4Error}` instead of `any`

Reply:
It causes the same problems as exists change

Reply:
You can do the same thing here as above.